ansible/roles/shd-client-core/templates/configmap.yml.j2

```diff
@@ -0,0 +1,13 @@
+#jinja2: trim_blocks: "True", lstrip_blocks: "True"
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: shd-client-configmap
+  namespace: {{ NAMESPACE }}
+  labels:
+    app: shd-client
+data:
+  # general
+  NODE_ENV: "production"
```

Comment:
Where is it used?

Reply:
It is used in the vue.config.js but I think that one is controlled by the Dockerfile during build, so I will remove it here

Reply:
Do I need TZ? I have looked into the other ansibles and they all have NODE_ENV and TZ.

Reply:
It was not about TZ it was about the NODE_ENV.
The reason why i ask if a veu/ts/js app who is put after build into an nginx really need NODE_ENV?

Reply:
Top, has not seen that you removed it in your last commit :)